gamestonk_terminal/forex/fx_controller.py

```diff
@@ -191,44 +179,27 @@ def call_news(self, other_args: List[str]):
         """Call news [News API]"""
         news_view.news(other_args, self.instrument)
 
-    def call_bullbear(self, other_args):
+    def call_bullbear(self, other_args: List[str]):
```

Comment:
Would it be possible to do the same as you did for "EDA" with "BA" ?

Reply:
I left news and reddit since they're from dd. Most of the dd menu doesn't work with forex, so it won't integrate the same way.